zipline/data/loader.py

```diff
@@ -249,6 +249,7 @@ def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day):
         data.to_csv(path)
     except (OSError, IOError, HTTPError):
         logger.exception('failed to cache the new benchmark returns')
+        raise
```

Comment:
it was, I think we don't want to put the `get_benchmark_returns` call in the same try except. If `data` cannot be read we need to fail but if we cannot cache it to disk for later runs we can continue.